src/main/update_installer.js

```diff
@@ -3,13 +3,15 @@ import phin from "phin";
 const semverGreaterThan = require("semver/functions/gt");
 const {version} = require("../../package.json");
 
+
```

Comment:
Please remove unnecessary additional newlines

Reply:
The dual newlines between functions was added for my own sanity as a contributor. Dual newlines between functions are very common in programming, to make large files easier to read, and to provide clear breaks between every function. Remember that your code is meant to be readable by every contributor, and this change helps that. I would like to formally request that dual newlines between functions stays. If that is okay with you. :)

Reply:
Yes, it is not up to contributors to determine the styleguide of a codebase. Plus doing both mass style and functional changes in one PR is out of scope imo.